tribits/ci_support/CDashQueryAnalyzeReport.py

```diff
@@ -561,9 +561,38 @@ def stripWhiltespaceFromStrList(strListInOut):
   for i in range(len(strListInOut)): strListInOut[i] = strListInOut[i].strip()
 
 
+g_expectedBuildsCsvFileHeadersRequired = \
+  ('group', 'site', 'buildname')
+
+
 def getExpectedBuildsListfromCsvFile(expectedBuildsFileName):
   return readCsvFileIntoListOfDicts(expectedBuildsFileName,
-    ['group', 'site', 'buildname'])
+    g_expectedBuildsCsvFileHeadersRequired)
+
+
+# Write list of builds from a builds LOD to a CSV file structure meant to
+# match the expected builds CSV file.
+#
+def expectedBuildsLODToCsvFileStructure(buildsLOD):
```

Comment:
"LOD" stands for "List Of Dicts" is a very common acronym used throughout this code.  Since Python is dynamically typed, it is sometimes nice to state the structure of the in a static way.  But generally LOD is spelled out in the function names as `ListOfDicts` so this is not consistent.  Having 'LOD' in the variable names makes more sense.  I will work to fix that.